app/models/item.rb

```diff
@@ -4,5 +4,9 @@ class Item < ApplicationRecord
   has_many :users, through: :bookings
   validates_presence_of :name, :description, :size, :price, :category
   CATEGORIES = ["bags", "dresses", "shoes"]
+  SIZES = ["XXS", "XS", "S", "M", "L", "XL", "XXL", "XXXL", "XXXXL"]
+  validates :size, inclusion: { in: SIZES }
```

Comment:
Maybe no validation for size because a bag for example does not have a size like s, m, l